README.md

```diff
@@ -4,63 +4,51 @@ Welcome to the home page for the MessageFormat Working Group, a subgroup of the
 
 ## Charter
 
-The MessageFormat Working Group (MFWG) is tasked with developing an industry standard
-for the representation of localizable message strings to be a successor to 
-[ICU MessageFormat](https://unicode-org.github.io/icu/userguide/format_parse/messages/). 
-MFWG will recommend how to remove redundancies, 
-make the syntax more usable, 
-and support more complex features, such as gender, inflections, and speech. 
-MFWG will also consider the integration of the new standard with programming environments, 
-including, but not limited to, ICU, DOM, and ECMAScript, and with localization platform interchange. 
-The output of MFWG will be a specification for the new syntax.
-
-- [Why ICU MessageFormat Needs a Successor](docs/why_mf_next.md)
-- [Goals and Non-Goals](docs/goals.md)
-
-## MessageFormat 2 Final Candidate
-
-The [MessageFormat 2 specification](./spec/) is a new part of
+The MessageFormat Working Group (MFWG) is tasked with developing and supporting an industry standard
+for the representation of localizable message strings. 
+MessageFormat is design to support software developers, translators, and end users with fluent messages
+and locally-adapted presentation of data values
+while developing support for increasingly complex features, such as gender, inflections, and speech. 
```

Comment:
Not really sure of what to make of the ending with "and speech", which feels like it's missing something?

```suggestion
MessageFormat is designed to support software developers, translators, and end users with fluent messages
and locally-adapted presentation for data values
while providing a framework for increasingly complex features, such as gender, inflections, and speech. 
```